debugger/main.ml

```diff
@@ -182,6 +184,16 @@ let speclist = [
       " Print version and exit";
    "-vnum", Arg.Unit print_version_num,
       " Print version number and exit";
+   "-no-version", Arg.Clear Parameters.version,
+      " Do not print version at startup";
+   "-no-prompt", Arg.Clear Parameters.prompt,
+      " Suppress all prompts";
+   "-no-time", Arg.Clear Parameters.time,
+      " Do not print times";
+   "-no-breakpoint", Arg.Clear Parameters.breakpoint,
```

Comment:
Could you change the option to `-no-breakpoint-message` ? I think it would be less misleading.